tests/test_common_semantic_segmentation_format.py

```diff
@@ -0,0 +1,129 @@
+import os.path as osp
+from collections import OrderedDict
+from unittest import TestCase
+
+import numpy as np
+
+from datumaro.components.annotation import Mask
+from datumaro.components.dataset import Dataset
+from datumaro.components.environment import Environment
+from datumaro.components.extractor import DatasetItem
+from datumaro.components.media import Image
+from datumaro.plugins.common_semantic_segmentation_format import (
+    CommonSemanticSegmentationImporter,
+    make_categories,
+)
+from datumaro.util.test_utils import compare_datasets
+
+from .requirements import Requirements, mark_requirement
+
+DUMMY_DATASET_DIR = osp.join(
+    osp.dirname(__file__), "assets", "common_semantic_segmentation_dataset", "dataset"
+)
+
+DUMMY_DATASET_WITH_NON_STANDARD_STRUCTURE_DIR = osp.join(
```

Comment:
This dataset has custom filenames, but the structure is the same as in the "normal" case. I suggest to rename this dataset to make the difference more clear.

Reply:
Renamed